plugins/Satellites/src/Satellites.hpp

```diff
@@ -74,7 +75,10 @@ By convention, group names are in lowercase. The GUI translates some of the grou
 
 <b>Satellite Catalog</b>
 
-The satellite catalog is stored on disk in [JSON](http://www.json.org/) format, in a file named "satellites.json". A default copy is embedded in the plug-in at compile time. A working copy is kept in the user data directory. Test for GitHub diff, are lines now soft wrapped?
+The satellite catalog is stored on disk in [JSON](http://www.json.org/) format,
+in a file named "satellites.json". A default copy is embedded 
+in the plug-in at compile time. A working copy is kept in the
+user data directory. Test for GitHub diff, are lines now soft wrapped?
```

Comment:
Lines are not soft-wrapped, but line length is OK now. 

This branch would have been almost done according to my understanding, only these minor issues were open. I don't know how much time it took to research the counterexamples of code that breaks the line length style rules, and I did not check all of these entries. Still, when reviewers insist on these things with new code from external contributors, there is no need to get upset on such minor details.

Reply:
that's exactly the point I wanted to make: it took me only a few minutes to find those examples (I just browsed the first PRs that came up), and **many** more could be pointed out. but I'm more than fed up with all these stupid remarks and daft counter arguments that basically stem from irrational attitudes and broken English and that bypass far more important issues such as improving software engineering practices, given for free from someone who has years of experience (and believes in and has practiced the Agile approach).

sorry guys, but that's NOT how you treat contributors. **and it's not the first time I point this out**,